packages/sdk/src/types/ApiLimits.ts

```diff
@@ -0,0 +1,4 @@
+export interface ApiLimits {
+	rateLimit: number | boolean
```

Comment:
I am not in favor of having a type where it can be both number and boolean since both `0` and `false` are `falsy` values and writing code like that:
```
if (!xyz.rateLimit)
```

or
```
if (xyz.rateLimit)
```

might be error prone.
In my opinion it would be better to have separate method:
`hasAnyRequestBeenMade` returning false/true and having `ApiLimits` with strict `number` type and highlight in documentation that developer should first make a call to `hasAnyRequestBeenMade` and then call to `ApiLimits`